core/src/main/scala/io/qbeast/core/transform/StringHistogramTransformer.scala

```diff
@@ -0,0 +1,45 @@
+package io.qbeast.core.transform
+
+import io.qbeast.core.model.QDataType
+import io.qbeast.core.transform.StringHistogramTransformer.{defaultHist, defaultHistStr}
+
+object StringHistogramTransformer extends TransformerType {
+  override def transformerSimpleName: String = "string_hist"
+
+  // "a" to "z"
+  val defaultHist: IndexedSeq[String] = (97 to 122).map(_.toChar.toString)
```

Comment:
Thinking if the defaultHist could be extracted from the data if the user allows it so...

Reply:
For next steps.